search-app/src/cmr/search/api/concepts_lookup.clj

```diff
@@ -73,13 +73,13 @@
 (defn- find-concept-by-concept-id*
   "Perfrom the retrieval of concept by concept id and revision id"
   ([ctx result-format concept-id]
-   (info (format "Search for concept with cmr-concept-id [%s]" concept-id))
+   (debug (format "Search for concept with cmr-concept-id [%s]" concept-id))
```

Comment:
Putting this comment here but really it applies to most of these changes.

Removing the start log is an interesting approach. I _think_ that we don't use them in dashboards, reports, etc - a quick look through some dashboards showed that we use the "found x <concept> in y ms" logs. We need to double, triple check that that's true before merging this.  I am a little bit concerned about losing end-to-end visibility of a searches lifecycle, but that might be fine.

Reply:
IIRC the "found x in y ms" logs only get created when results are found. So searches that don't find anything or error out won't get captured

Reply:
But I think the access logs should cover it

Reply:
I went through the Splunk dashbaords and alerts and did not find any query that uses 'Search for concept...' I only found the one using 'Searching for...'

Ad for the actual info, this is already captured in the REPORT log

Reply:
> IIRC the "found x in y ms" logs only get created when results are found. So searches that don't find anything or error out won't get captured

Yeah, that's what my comment was saying - we don't use these for reports. IIRC, john had some reason to not get rid of this particular log though, right?

Reply:
I will put these logs back